rcl_interfaces/msg/IntegerRange.msg

```diff
@@ -1,15 +1,13 @@
 # Represents bounds and a step value for an integer typed parameter.
 
 # Lower bound for valid values, inclusive.
```

Comment:
Similar comment here, I think left and right are appropriate because we are speaking about a "number line" (https://en.wikipedia.org/wiki/Number_line).